pkg/api/v1alpha1/addons_types.go

```diff
@@ -230,14 +204,18 @@ const (
 
 	// AddonsLayerHoldMsg explains the hold status.
 	AddonsLayerHoldMsg string = ("AddonsLayer custom resource has the 'Hold' element set to ture" +
-		"preventing it from being processed. To clear this state update the custom resource object" +
-		"setting Hold to false")
+		" preventing it from being processed. To clear this state update the custom resource object" +
+		" setting Hold to false")
+
+	// AddonsLayerDeployedReason represents the fact that the addons has been successfully deployed.
+	AddonsLayerDeployedReason string = "AddonsLayer is Deployed"
 )
 
 // AddonsLayer is the Schema for the addons API.
-// +genclient
-// +genclient:Namespaced
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster
```

Comment:
Might also be worth adding some `// +kubebuilder:printcolumn:name=` statements as this really helps when someone does `kubectl get addonslayers`

Reply:
We have an issue in backlog for this https://github.com/fidelity/kraan/issues/13

Reply:
Awesome